int/api/server.go

```diff
@@ -60,10 +56,23 @@ func parseNetworkFlag(massaNodeServerPtr *string) {
 	os.Setenv("MASSA_NODE_URL", *massaNodeServerPtr)
 }
 
+func stopServer(app *fyne.App, server *restapi.Server, manager *pluginmanager.PluginManager) {
+	defer func() {
```

Comment:
I think the "defer" is useless here.. just stops the server directly

Reply:
defer is used to set a callback when the function returns. here your fuction does not do anything else

Reply:
I disagree, see my comment, defer is useful to stop the server if the code panic while handling an error

Reply:
Oh okay. Then I will do as you said in your comment. Thanks a lot !